apps/poc-next/app/me/page.tsx

```diff
@@ -20,36 +20,29 @@ const Account = () => {
   const [shownAddress, setShownAddress] = useState("No address found")
 
   useEffect(() => {
-    if (!token || !userId) {
-      router.push(siteConfig.links.signIn)
-    } else {
-      ;(async () => {
-        try {
-          if (address) {
-            const prefix = address.replace(/(.{5})..+/, "$1…")
-            const postfix = address.substring(
-              address.length - 4,
-              address.length
-            )
-            setShownAddress(prefix + postfix)
-          }
-          const { user } = await fetch(
-            `${process.env.NEXT_PUBLIC_API_URL}/user/${userId}`,
-            {
-              method: "GET",
-              headers: {
-                "Content-Type": "application/json",
-                Authorization: `Bearer ${token}`,
-              },
-            }
-          ).then((res) => res.json())
-          setSubmittedURLs(user.submittedUrls.length)
-          setLikedURLs(user.likedUrls.length)
-        } catch (error) {
-          console.log(error)
+    ;(async () => {
```

Comment:
whats this `;` do?

Reply:
I am not sure, it was added by the prettier/es-lint thing